lib/src/views/chatroom_page.dart

```diff
@@ -69,6 +72,9 @@ class _ChatRoomPageState extends State<ChatRoomPage> {
   ValueNotifier rebuildConversationList = ValueNotifier(false);
   ValueNotifier rebuildChatBar = ValueNotifier(false);
   ValueNotifier showConversationActions = ValueNotifier(false);
+  ValueNotifier<bool> rebuildChatTopic = ValueNotifier(true);
+  bool showChatTopic = true;
+  Conversation? tmpTopic;
```

Comment:
If we can change the variable name `tmpTopic` to something more self explanatory it would be good, just a suggestion though

Reply:
changing `tmpTopic` to `localTopic`

Reply:
Okay